microprofile-reactive-messaging-kafka/src/main/java/org/jboss/as/quickstarts/reactive/messaging/DatabaseBean.java

```diff
@@ -0,0 +1,31 @@
+package org.jboss.as.quickstarts.reactive.messaging;
```

Comment:
No need to use legacy package - s/org.jboss.as.quickstarts/org.wildfly.quickstarts

Reply:
Moreover, for MP we are using ` org.wildfly.quickstarts.microprofile` package.